web-config.yaml

```diff
@@ -9,17 +11,18 @@ web:
         ttl: 3600
     password:
       enabled: true
+      validationStrategies: ['local','stormpath']
   accessTokenCookie:
     name: "access_token"
     httpOnly: true
     secure: null
-    path: "/"
+    path: null
```

Comment:
Interesting - you already made the path change here.  I suspect this is the current incarnation `path: null` and the above was just out of date?